src/renderer/components/NodeSelectorPanel/NodeSelectorPanel.tsx

```diff
@@ -34,11 +35,44 @@ import {
 } from '../../helpers/nodeSearchFuncs';
 import { useNodeFavorites } from '../../hooks/useNodeFavorites';
 import { FavoritesAccordionItem } from './FavoritesAccordionItem';
-import { RegularAccordionItem } from './RegularAccordionItem';
+import { PackageHint, RegularAccordionItem, Subcategories } from './RegularAccordionItem';
 import { TextBox } from './TextBox';
 
+const packageHints: Partial<Record<string, () => ReactNode>> = {
```

Comment:
Instead of defining these here, could we define them in dependencies.js? This doesn't seem very DRY. Then we could just build each one of these from the information there

Reply:
Because dependencies.ts is in src/common and that doesn't do React.

Reply:
Hints are a frontend thing.

Reply:
Oh right I forgot this uses the categories from the backend. Maybe we could define this stuff there? I don't mean defining the react stuff but defining the strings used, and then just {description} them, ya know. That can be handled in a different ticket though.